firebase-firestore/src/test/java/com/google/firebase/firestore/model/FieldValueTest.java

```diff
@@ -411,7 +411,7 @@ public void testArraysFail() {
       wrap(array);
       fail("wrap should have failed");
     } catch (IllegalArgumentException e) {
-      assertNotEquals(-1, e.getMessage().indexOf("use a List instead"));
+      assertNotEquals(-1, e.getMessage().indexOf("use Lists instead"));
```

Comment:
This error is now coming from CustomClassMapper and so the wording changed slightly.